index.html

```diff
@@ -332,12 +332,7 @@ <h3 id="unacceptablebehavior">
           violated this code of conduct.
           </li>
         </ul>
-        <p>
-          This Code prioritizes the safety of individuals within marginalized
-          communities over the comfort of others, and those enforcing it
-          reserve the right not to take further actions on complaints
-          regarding:
-        </p>
+        <p>This Code prioritizes the safety of individuals over the comfort of others. In situations where there is a balance of factors to consider, greater weight will be given to protecting those experiencing less privilege; for example:</p>
```

Comment:
As phrased, there's something missing after "for example", e.g. "the code does not recognize complaints such as the following:"

Reply:
I agree here, the current edit weakens the statement, it needs to make it clear that some complaints will not receive action.

Reply:
There's no reason every complaint shouldn't receive an action, even if that action is to dismiss the complaint as invalid. We [formally address](https://www.w3.org/2019/Process-20190301/#formal-address) every issue against our specs, accepting it if the WG agrees to fix, or rejecting it (with rationale) if it's invalid or inappropriate to address; I don't see why this system ought to operate at a lesser level of integrity.

Reply:
side comment: https://rhodesmill.org/brandon/2012/one-sentence-per-line/ helps with reading diffs :) :)

Reply:
Agree with @wseltzer suggestion. 

(@fantasai as an aside, I had to "html tidy" up the doc because it was previously quite messy with broken markup... Although the one sentence per line thing is really good practice, particular for markdown/Word/text/wysiwyg docs, it's not super good for markup - I'll help those editing this doc use HTML Tidy, making it easier to see the diffs properly).